resources/function_help/json/regexp_matches

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "regexp_matches",
+  "type": "function",
+  "description": "Returns an array of all strings captured by capturing groups, in the order the groups themselves appear in the supplied regular expression against a string.<br>If matches are found, the first element of the returned array will contain the text that mathed the full regular expression pattern.",
+  "arguments": [
+    {"arg":"string", "description":"the string to capture groups from against the regular expression"},
+    {"arg":"regex","description":"the regular expression used to capture groups"},
+    {"arg":"emptyvalue","optional":true,"default":"''","description":"the optional string to use as replacement to empty array values"}],
+  "examples": [ { "expression":"regexp_matches('qgis=>rocks;hello=>world','qgis=>(.*)[;$]')", "returns":"array: 'qgis=>rocks;', 'rocks'"}
```

Comment:
Can you add an example with the empty value parameter? I don't find this parameter to be obvious and wonder if it is just cluttering this function.